### ISSUE TITLE
Way to access generated entrypoint chunks without index.html parsing

### ISSUE THREAD

**Issue by lifeart** [ef4/ember-auto-import#472]
There a list of Ember use-cases without original index.html file usage (like app embedding), and with v2 approach, there is no way to sort js/css imports except analyzing dist/index.html

For this purposes, we could use `Inserter. categorizeChunks()` method, but it's not public

https://github.com/peopledoc/ember-auto-import-chunks-json-generator

addon created to get access to it, but in long term, in my understanding, `auto-import` should handle it automatically, and generate kinda "ordered-importmap" in place with index.html

**Comment by simonihmig** [comment]
FWIW, this use case of embedding apps is something we deal a lot with, and the addon [ember-embedded-snippet](https://github.com/kaliber5/ember-embedded-snippet) we have for this does exactly this: it parses index.html after the actual build. (changed that more recently to support eai v2 and Embroider, as we cannot assume anymore an app just consists of `vendor.js` and `app.js`)

It does have the added benefit that it is able to see any other asset not added only by eai. For example our [ember-responsive-image](https://github.com/kaliber5/ember-responsive-image) adds a json script to index.html, which the embedded addon is now also capable of capturing! That wouldn't have been the case with your proposed chunk map.

**Comment by ef4** [comment]
Yes, I am in favor of continuing to use the HTML as the authoritative source. A big reason for this is that it's robust against many future changes to how apps get packaged and distributed in a way that other formats are not.
